db/post_migrate/20231120190818_trigger_post_rebake_category_style_quotes.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class TriggerPostRebakeCategoryStyleQuotes < ActiveRecord::Migration[7.0]
+  def up
+    DB.exec(<<~SQL)
+      UPDATE posts
+      SET baked_version = NULL
+      WHERE cooked LIKE '%blockquote%'
+    SQL
+  end
+
+  def down
+    raise ActiveRecord::IrreversibleMigration
```

Comment:
I would not call this one irreversible, simply not immediately reversible. However, we do reverts so rarely that it does not matter that much.